apps/docs/content/components/CheckboxGroup/react.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Checkbox group
+description: Checkbox group renders a set of checkboxes.
+source: https://github.com/primer/brand/blob/main/packages/react/src/forms/CheckboxGroup/CheckboxGroup.tsx
+figma: 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=10467-3273&t=GA5GtEqPtQ9yeRaN-4'
+storybook: '/brand/storybook/?path=/story/components-forms-checkboxgroup--playground'
+---
+
+import {Label} from '@primer/react'
+import {PropTableValues} from '../../../src/components'
+
```

Comment:
Tabs aren't working in the React docs, making it hard to go back to overview. Think you're just missing this:

```suggestion

import ComponentLayout from '../../../src/layouts/component-layout'
export default ComponentLayout

```

Issue:


https://github.com/user-attachments/assets/afd35600-9ee8-45be-a394-6cfbf8f77e89

Reply:
Well spotted, thank you